pywbem_mock/_wbemconnection_mock.py

```diff
@@ -1068,124 +1300,91 @@ def _display_objects(obj_type, object_repo, namespace, cmt_begin, cmt_end,
 
-        if not namespaces:
-            namespaces = [self.default_namespace]
+            If `None`, the default namespace of the connection will be used.
 
-            if not isinstance(namespaces, (list, tuple, six.string_types)):
-                raise TypeError(
-                    _format('Namespace argument {0|A} must be a string or '
-                            'list/tuple but is {1}',
-                            namespaces, type(namespaces)))
+          schema_pragma_files  (:term:`string` or :class:`py:list` of :term:`string`):
+            File paths defining a schema pragma file MOF for the set of CIM
+            classes that make up a schema such as the DMTF schema. These files
+            must contain include pragams defining the file location of the
+            classes to be compiled for the defined provider and for any
+            dependencies required to compile those classes.  The directory
+            containing each pragma file is passed to the MOF compiler as the
+            search path for compile dependencies.
 
-            if isinstance(namespaces, six.string_types):
-                namespaces = [namespaces]
+            See :class:`pywbem.MofCompiler` for more information on the
+            `search_paths` parameter.
 
-            for namespace in namespaces:
-                if namespace not in self.namespaces:
-                    raise ValueError(
-                        _format('Namespace "{0!A}" in namespaces argument not '
-                                'in CIM repository. '
-                                'Existing namespaces are: {1!A}. ',
-                                namespace, self.namespaces))
-
-        # If classnames is list, recursively call this method
-        if isinstance(classnames, (list, tuple)):
-            for classname in classnames:
-                self.register_provider(namespaces, classname, provider_type,
-                                       provider)
-            return
+          verbose ():
+            Flag to enable detailed display of actions
 
-        # Processing for classnames defining a single classname
-        classname = classnames
-        assert isinstance(classname, six.string_types)
+        Raises:
+            TypeError: Invalid provider_type retrieved from provider or
+                       provider_type does not match superlclass or the
+                       namespace parameter is invalid.
 
-        if isinstance(namespaces, six.string_types):
-            namespaces = [namespaces]
-        for namespace in namespaces:
-            if not self.mainprovider.class_exists(namespace, classname):
-                raise ValueError(
-                    _format('class "{0!A}" does not exist in '
-                            'namespace {1!A} of the CIM repository',
-                            classname, namespace))
+.           ValueError: Provider_type retrieved from provider is not a
```

Comment:
Remove the dot at the begin of the line. Sphinx gets disturbed by it.

Reply:
DONE